src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -3,53 +3,56 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls">
 
-    <x:Double x:Key="StandardFontSize">15.0</x:Double>
-    <Thickness x:Key="StandardIndentMargin">20,0,0,0</Thickness>
-    <Thickness x:Key="StandardControlSpacing">0,0,0,20</Thickness>
+    <x:Double x:Key="StandardIconSize">14.0</x:Double>
+    <Thickness x:Key="StandardIndentMargin">13,0,0,0</Thickness>
     <x:Double x:Key="StandardBoxMinWidth">250</x:Double>
-    
-    <Thickness x:Key="PivotIndentMargin">10,0,0,0</Thickness>
-    <Thickness x:Key="PivotStackPanelMargin">0,10,0,0</Thickness>
 
     <!-- This is for easier transition to the SettingsContainer control.
          The SettingsContainer will wrap a setting with inheritance UI.-->
     <Style x:Key="SettingContainerStyle" TargetType="ContentPresenter">
-        <Setter Property="Padding" Value="{StaticResource StandardControlSpacing}"/>
+        <Setter Property="Margin" Value="0,24,0,0"/>
+    </Style>
+
+    <!-- This is for styling the entire items control used on the
+         color schemes page-->
+    <Style x:Key="ItemsControlStyle" TargetType="ItemsControl">
+        <Setter Property="Margin" Value="0,24,0,0"/>
     </Style>
 
     <!--Used to stack a group of settings-->
     <Style x:Key="SettingsStackStyle" TargetType="StackPanel">
         <Setter Property="HorizontalAlignment" Value="Left"/>
-        <Setter Property="Margin" Value="{StaticResource StandardIndentMargin}"/>
+        <Setter Property="Margin" Value="13,0,0,48"/>
+    </Style>
+
+    <!--Used to stack a group of settings inside a pivot-->
+    <Style x:Key="PivotStackStyle" TargetType="StackPanel">
+        <Setter Property="HorizontalAlignment" Value="Left"/>
+        <Setter Property="Margin" Value="0,0,0,48"/>
     </Style>
 
     <!--Radio Button-->
     <Style x:Key="RadioButtonsSettingStyle" TargetType="muxc:RadioButtons">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>

```

Comment:
If we're relying on the defaults from WinUI, maybe we should remove these `ToolTipService.Placement` too. At that point, hate to say it, but we'd have some styles that don't hold anything inside of them, so we should remove them too.